recipes/ogdf/all/conanfile.py

```diff
@@ -0,0 +1,101 @@
+from conan import ConanFile
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import collect_libs, copy, get, replace_in_file, rmdir
+from conan.tools.microsoft import is_msvc
+from os.path import join
+
+required_conan_version = ">=1.53.0"
+
+
+class OGDFConan(ConanFile):
+    name = "ogdf"
+    description = "Open Graph algorithms and Data structures Framework"
+    license = "GPL-2.0-or-later"
```

Comment:
```suggestion
    license = ("GPL-2.0-or-later", "GPL-3.0-or-later")
```
There are 2 types of license available. https://github.com/ogdf/ogdf/blob/master/LICENSE.txt